clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/model/ServerGroup.groovy

```diff
@@ -122,6 +123,7 @@ interface ServerGroup {
   @JsonIgnore
   ImageSummary getImageSummary()
 
+  @JsonPropertyOrder(alphabetic = true)
```

Comment:
Nice - this will also help with eTags (which we used to provide in responses, not sure if we still do or not).